src/rgw/rgw_admin.cc

```diff
@@ -239,6 +239,8 @@ void usage()
   cout << "  metadata list              list metadata info\n";
   cout << "  mdlog list                 list metadata log\n";
   cout << "  mdlog trim                 trim metadata log (use marker)\n";
+  cout << "  mdlog autotrim             auto trim metadata log\n";
+  cout << "  mdlog fetch                fetch metadata log\n";
```

Comment:
i'm not sure where `mdlog fetch` came from. i see a command for it, but there's no actual implementation:

```
$ grep MDLOG_FETCH rgw_admin.cc
  MDLOG_FETCH,
  { "mdlog fetch", OPT::MDLOG_FETCH },
```